src/table/operations.jl

```diff
@@ -105,7 +133,33 @@ function reduce(f, d::DTable; cols=nothing::Union{Nothing, Vector{Symbol}}, init
     Dagger.@spawn construct_result(columns, reduce_chunks...)
 end
 
+"""
+    reduce(f, gd::GDTable; cols=nothing, [init]) -> NamedTuple
+
+Reduces `gd` using function `f` applied on all columns of the DTable.
+Returns results per group in columns with prefix `result_*`.
```

Comment:
Hmm, it will not be reused, so not sure if a warning is necessary
I mean the column will be called "result_result_*" after execution so it's just untidy

Reply:
Fair point!